### PR TITLE
Print error when APIServer fails to start

### PR DESCRIPTION
**Pull request by apelisse**
**What this PR does / why we need it**: Print apiserver logs when it fails to start.
This is copied from `update-swagger-spec.sh`.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: hack/update-openapi-spec.sh [hunk 1]
```diff
@@ -53,6 +53,7 @@ ETCD_HOST=${ETCD_HOST:-127.0.0.1}
 ETCD_PORT=${ETCD_PORT:-2379}
 API_PORT=${API_PORT:-8050}
 API_HOST=${API_HOST:-127.0.0.1}
+API_LOGFILE=/tmp/openapi-api-server.log
```
Comment (fejta) [inline hack/update-openapi-spec.sh]: Do we not want to do the same `${FOO:-default}` pattern the rest use?

**Review by fejta — COMMENTED** [review]
/retest
/lgtm
/hold

File: hack/update-openapi-spec.sh (hunk 1 above)
Comment (apelisse) [reply hack/update-openapi-spec.sh]: Configuring host/port for etcd and apiserver let's one run against an existing server. I don't think we care for the logfile.

/hole cancel

Comment (apelisse) [reply hack/update-openapi-spec.sh]: I ended-up updating :-) Thanks!

**Pull request merged**